php/class-coauthors-guest-authors.php

```diff
@@ -1503,4 +1506,93 @@ public function filter_author_feed_link( $feed_link, $feed ) {
 
 		return $link;
 	}
+
+	/**
+	 * Filter Personal Data Exporters to add Guest Author exporter
+	 *
+	 * @since 3.3.1
+	 */
+    public function filter_personal_data_exporter( $exporters ) {
+        $exporters['cap-guest-author'] = array(
+            'exporter_friendly_name' => __( 'Guest Author', 'co-authors-plus' ),
+            'callback'               => array( $this, 'personal_data_exporter' ),
+        );
+
+        return $exporters;
+    }
+
+	/**
+	 * Finds and exports personal data associated with an email address for guest authors
+	 *
+	 * @since 3.3.1
+	 *
+	 * @param string $email_address  The guest author email address.
+ 	 * @return array An array of personal data.
+	 */
+	public function personal_data_exporter( $email_address ) {
+		$email_address = trim( $email_address );
+
+		$data_to_export = array();
+
+		$author = $this->get_guest_author_by( 'user_email', $email_address );
+
+		if ( ! $author ) {
+			return array(
+				'data' => array(),
+				'done' => true,
+			);
+		}
+
+		$author_data = array(
+			'ID'              => __( 'Guest Author ID', 'co-authors-plus' ),
```

Comment:
I think we can remove `Guest Author` from the labels here as it's a bit redundant.

Reply:
I added it because user export has "User" label as well. But I agree it's redundant.